corpus/accounts/models.py

```diff
@@ -2,7 +2,7 @@
 from django.contrib.auth.base_user import BaseUserManager
 from django.contrib.auth.models import AbstractUser
 from django.db import models
-from django.utils import timezone
+from django.utils.timezone import now
```

Comment:
Import timezone, and use the function like that. This is a kinda weird way to use it xD

```
from django.utils import timezone

timezone.now()
```